src/expr/src/expr/build.rs

```diff
@@ -42,11 +42,13 @@ use crate::{bail, ExprError, Result};
 pub fn build_from_prost(prost: &ExprNode) -> Result<BoxedExpression> {
     use PbType as E;
 
+    let binding = FunctionCall { children: vec![] };
     let func_call = match prost.get_rex_node()? {
         RexNode::InputRef(_) => return InputRefExpression::try_from_boxed(prost),
         RexNode::Constant(_) => return LiteralExpression::try_from_boxed(prost),
         RexNode::Udf(_) => return UdfExpression::try_from_boxed(prost),
         RexNode::FuncCall(func_call) => func_call,
+        RexNode::Now(_) => &binding,
```

Comment:
It seems this branch is unreachable since we're not going to execute `Now` in the backend. What about simply `unreachable!()` here?